draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1382,16 +1366,24 @@ Disclosures:
 
 {{examples/w3c-vc/disclosures.md}}
 
-## Example 4b - W3C Verifiable Credentials Data Model v2.0, using JSON-LD
+A presentation of the SD-JWT that discloses only `given_name` and `is_over_18`
+claims with a Key Binding JWT could look as follows:
 
-This example illustrates how to use the artifacts defined in this specification to secure a payload
-that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
-and uses a JSON-LD object as the claims set. The example uses a content type `credential+ld+json` defined in [@VC_DATA_v2.0], Section 6.3 in a `cty` JOSE Header value.
+<{{examples/w3c-vc/sd_jwt_presentation.txt}}
 
-SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a VC.
+The payload of a corresponding Key Binding JWT looks as follows:
 
-An SD-JWT Presentation with a Key Binding JWT would be equivalent to a Holder-signed
-Verifiable Presentation as defined in [@VC_DATA_v2.0].
+<{{examples/w3c-vc/kb_jwt_payload.json}}
+
+After the verification of the data, the Verifier will
+pass the following result on to the application for further processing:
+
+<{{examples/w3c-vc/verified_contents.json}}
+
+## Example 4b - W3C Verifiable Credentials Data Model v2.0
+
+This example illustrates how to use the artifacts defined in this specification to secure a payload
+that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]. The example uses a media type `vc+ld+json` defined in [@VC_DATA_v2.0] Section 6.1 in a `typ` JOSE Header value.
```

Comment:
The example doesn't actually have a `typ` JOSE header parameter.  And I think `vc+ld+json` would be used in the `cty` header. Anyway, as mentioned in another comment, I think these examples might be better off just not saying anything about typ or cty headers or media types.



```suggestion
that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0].
```